internal/compiler/builtins.slint

```diff
@@ -217,8 +217,8 @@ export component TextInput {
     in property <bool> font-italic;
     in property <int> font-weight;
     in property <brush> color; // StyleMetrics.default-text-color  set in apply_default_properties_from_style
```

Comment:
```suggestion
    in property <brush> color; // Palette.default-text-color  set in apply_default_properties_from_style
```

If you're updating the code to use the palette, this comment needs to be updated.